test/e2e/system_df_test.go

```diff
@@ -48,14 +48,19 @@ var _ = Describe("podman system df", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
 
+		session = podmanTest.Podman([]string{"images", "-q"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		totImages := strconv.Itoa(len(session.OutputToStringArray()))
```

Comment:
Thank you! That has bitten us before.

Purely out of curiosity, is it ever possible for this number to be different from `podman info --format '{{.Store.ImageStore.Number}}'`?